src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/BlobProcessor.java

```diff
@@ -70,7 +70,8 @@ public void process(BlobClient blobClient) {
             envelopeSupplier = () -> envelopeService.createNewEnvelope(
                 blobClient.getContainerName(),
                 blobClient.getBlobName(),
-                blobClient.getProperties().getCreationTime().toInstant()
+                blobClient.getProperties().getCreationTime().toInstant(),
+                blobClient.getProperties().getBlobSize()
```

Comment:
which one is correct `blobClient.getProperties().getBlobSize()` or `b.getProperties().getContentLength()` or